tensorflow/python/framework/tensor_util.py

```diff
@@ -981,7 +981,17 @@ def is_tensor(x):  # pylint: disable=invalid-name
 
   If `is_tensor(x)` returns `True`, it is safe to assume that `x` is a tensor or
   can be converted to a tensor using `ops.convert_to_tensor(x)`.
-
+  
+  Usage example:  
+    >>> x = tf.constant([[1,2,3],[4,5,6],[7,8,9]])
+    >>> print(x.shape)
+    (3, 3)
+    >>> print(tf.is_tensor(x))
+    True
+    >>> y = "Hello World"
+    >>> print(tf.is_tensor(y))
+    False
+    
```

Comment:
Hi, Please indent the code at the same level as the 'Usage Example:'

Reply:
I have updated the indentation as you requested and re-tested it locally.

Reply:
No need to use `print`, this is similar to an interactive session.

Also, no need to print the shape of the tensor.

How about

```python
>>> tf.is_tensor(tf.constant(...))
True
>>> tf.is_tensor("Hello World")
False
```

This should keep the example minimal, focused on exactly what it is documenting.

Reply:
I have edited the usage example as you requested and re-tested it locally.